### ISSUE TITLE
IdTable<int> Can't use not autoincrement id field

### ISSUE THREAD

**Issue by Aileron1410** [JetBrains/Exposed#57]
issue is only in 0.7.0, 0.6.8 is fine with it
Entity.kt 430
```
                for ((entry, genValues) in toFlush.zip(ids)) {
// genvalues are empty, null pointer Exception
                    val id = genValues[table.id]!!

// rewrites always by generateg value
                    entry.id._value = (table.id.columnType as EntityIDColumnType<*>).idColumn.columnType.valueFromDB(when (id) {
                        is EntityID<*> -> id._value!!
                        else -> id
                    })
```
Thank you!

**Comment by Tapac** [comment]
@Aileron1410 , can you provide simple snippet to reproduce that case and please ensure that it doesn't work on latest version ( 0.7.1 ).

**Comment by Tapac** [comment]
I can reproduce it by myself, so will fix it ASAP (and add code as test).
Thank you for report

**Comment by Tapac** [comment]
Fixed. Will be available a bit later in 0.7.2

**Comment by Aileron1410** [comment]
Thank you very much!
We will waight 0.7.2)

19 Окт 2016 г. 21:16 пользователь "Andrey.Tarashevskiy" <
notifications@github.com> написал:

**Comment by Tapac** [comment]
Fixed in 0.7.2 version

**Comment by Aileron1410** [comment]
Thank you very much, it works fine now
